e2e/cypress/integration/integrations/custom_slash_commands/slash_commands_spec.js

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Slash commands page', () => {
+    const trigger = 'test-message';
+    let testTeam;
+
+    before(() => {
+        cy.requireWebhookServer();
+    });
+
+    beforeEach(() => {
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Go to integrations
+            cy.visit(`/${team.name}/integrations`);
+
+            // * Validate that slash command section is enabled
+            cy.get('#slashCommands').should('be.visible');
+
+            // # Open slash command page
+            cy.get('#slashCommands').click();
+        });
+    });
+
+    it('MM-T690 Add custom slash command: / error', () => {
+        // # Add new command
+        cy.get('#addSlashCommand').click();
+
+        // # Type a trigger starting with slash
+        cy.get('#trigger').type('//input');
+
+        // # Save
+        cy.get('#saveCommand').click();
+
+        // * Verify that we get the error message
+        cy.findByText('A trigger word cannot begin with a /').should('exist').and('be.visible').scrollIntoView();
+    });
+
+    it('MM-T691 Error: trigger word required', () => {
+        // # Add new command
+        cy.get('#addSlashCommand').click();
+
+        // # Do not input trigger word
+        cy.get('#url').type('http://hidden-peak-21733.herokuapp.com/test_ephemeral');
```

Comment:
One last request. Could you change all reference to this URL, maybe use `http://example.com` instead?  Thanks